internal/node/node_loader.js

```diff
@@ -25,6 +25,14 @@
 var path = require('path');
 var fs = require('fs');
 
+const isWindows = /^win/i.test(process.platform);
+// Ensure that node is added to the path for any subprocess calls
+if (isWindows) {
+  process.env.PATH = `${path.dirname(process.execPath)};${process.env.PATH}`
```

Comment:
nit: less duplication with
```
process.env.PATH = `${path.dirname(process.execPath)}${isWindows ? ';' : ':'}${process.env.PATH}`;
```

Reply:
Or to avoid all the quotes and `${...}`...
```
[path.dirname(process.execPath), process.env.PATH].join(isWindows ? ';' : ':')
```